packages/@o3r/eslint-plugin/src/rules/yaml/yarnrc-package-extensions-harmonize/yarnrc-package-extensions-harmonize.ts

```diff
@@ -99,49 +106,48 @@ export default createRule<[YarnrcPackageExtensionsHarmonizeOptions, ...any], 've
     const ignoredDependencies = options.ignoredDependencies.map((dep) => new RegExp(dep.replace(/[$()+.?[\\\]^{|}]/g, '\\$&').replace(/\*/g, '.*')));
 
     if (parserServices.isYAML) {
-      const rule = (node: AST.YAMLPair) => {
-        if (node.value) {
-          const range = getStaticYAMLValue(node.value)?.toString();
-          const parent = node.parent.parent && node.parent.parent.type === 'YAMLPair' && getStaticYAMLValue(node.parent.parent.key!)?.toString();
-          const baseNode = node.parent.parent.parent.parent?.parent?.parent;
-          const isCorrectNode = baseNode && baseNode.type === 'YAMLPair' && getStaticYAMLValue(baseNode.key!)?.toString() === 'packageExtensions';
-          if (isCorrectNode && semver.validRange(range) && parent && options.yarnrcDependencyTypes.includes(parent)) {
-            const depName = node.key && getStaticYAMLValue(node.key)?.toString();
-            if (!depName || !bestRanges[depName] || ignoredDependencies.some((ignore) => ignore.test(depName))) {
-              return;
-            }
-            const minYarnrcVersion = semver.minVersion(range!);
-            const minBestRangeVersion = semver.minVersion(bestRanges[depName].range);
-            if (minYarnrcVersion && minBestRangeVersion && semver.lt(minYarnrcVersion, minBestRangeVersion)) {
-              const version = bestRanges[depName].range;
-              const packageJsonFile = bestRanges[depName].path;
-              context.report({
-                loc: node.value.loc,
-                messageId: 'error',
-                data: {
-                  depName,
-                  version,
-                  packageJsonFile
-                },
-                fix: (fixer) => fixer.replaceTextRange(node.value!.range, `${version}`),
-                suggest: [
-                  {
-                    messageId: 'versionUpdate',
-                    data: {
-                      version
-                    },
-                    fix: (fixer) => fixer.replaceTextRange(node.value!.range, `${version}`)
-                  }
-                ]
-              });
+      return {
+        // eslint-disable-next-line @typescript-eslint/naming-convention -- name required by Yaml Eslint parser
```

Comment:
do we still need that?

Reply:
Yes, it was still required
(I think we have a linter for the unneeded linter rules)

Reply:
Yes we have one linter option to detect un-needed disable